xml/article_sap_trento.xml

```diff
@@ -1064,6 +1064,50 @@ Warning: Permanently added '<replaceable>TRENTO_AGENT_IP</replaceable>' (ECDSA)
 
   </section>
 
+   <section xml:id="sec-trento-uninstall-trentoserver">
+    <title>Uninstalling &t.server;</title>
+    <para> To uninstall &t.server;, do the following: </para>
+    <procedure>
+      <step>
+        <para>
+          Ensure that credentials for Kubernetes are set up correctly.
+          For example, if you run K3s, export the <envar>KUBECONFIG</envar>
+          environment variable:
+        </para>
+        <screen>export KUBECONFIG=/etc/rancher/k3s/k3s.yaml</screen>
+      </step>
+      <step>
+        <para>
+          With the same user that installed &t.server; run the following command:
+        </para>
+        <screen>helm uninstall trento-server
+        </screen>
```

Comment:
I would prefer to avoid linebreaks (the `<screen>` is verbatim text, every whitespace is preserved):

```suggestion
        <screen>helm uninstall trento-server</screen>
```